packaging/ubuntu-14.04/rules

```diff
@@ -103,6 +103,8 @@ override_dh_auto_build:
 	cd cmd && ( autoreconf -i -f )
 	cd cmd && ( ./configure --prefix=/usr --libexecdir=/usr/lib/snapd $(VENDOR_ARGS))
 	$(MAKE) -C cmd all
+	# Build any available upstart configurations
+	$(MAKE) -C data/upstart all
```

Comment:
Aren't we doing that already? We just take the defaults here as we do similar for the systemd build step. This works similar like with default values for configure scripts.